contrib/python/src/python/pants/contrib/python/checks/checker/common.py

```diff
@@ -14,9 +14,16 @@
 from io import StringIO
 
 import six
-from pants.util.collections_abc_backport import Sequence
+from future.utils import PY3
 
 
+# NB: we cannot use `pants.util.collections_abc_backport` because this library
+# must exist entirely independently of Pants src code.
```

Comment:
FWIW - this is not true. It must transitively depend on code that is py2.py3 compatible. It just so happens pants is not currently _distributed_ in this way whether or not the underlying pants code is actually py2.py3 compatible, which it is or is close to.